mouse_test.go

```diff
@@ -1,4 +1,4 @@
-package uinput
+package main
```

Comment:
What's the reason behind this change of package? Is this intended or the result of an automatic refactoring performed by the IDE? If this change is intentional, it would be interesting to understand the reasoning behind it. Otherwise, it would be strongly preferred to keep this consistent with the rest of the package.